spec/std/big/big_decimal_spec.cr

```diff
@@ -220,7 +288,7 @@ describe BigDecimal do
     1.5.to_big_r.to_big_d.should eq(BigDecimal.new(15, 1))
   end
 
-  it "can be converted from scientific notation" do
+  it "can be converted from 'scientific' or 'E' notation" do
```

Comment:
Is this necessary?

Reply:
Can be reverted if too noisy, just think the use of the terminology of 'E notation' is probably how I think about `2.1e2` rather than scientific notation makes me think of `2.1 x 10^2` .. but both work here

Reply:
It's just a spec description, not a formal definition =)

Reply:
But specs are the best documentation ;) I'll revert those soon